api-schema/src/main/java/org/stellar/anchor/api/shared/Amount.java

```diff
@@ -12,4 +13,12 @@ public class Amount {
   String asset;
 
   public Amount() {}
+
```

Comment:
💡 You may use Lombok [@AllArgsConstructor](http://www.javabyexamples.com/delombok-allargsconstructor-noargsconstructor-and-requiredargsconstructor)

Reply:
https://github.com/stellar/java-stellar-anchor-sdk/blob/b22f7b54f11c0c81d515cc947aeae950d75ca8d5/api-schema/src/main/java/org/stellar/anchor/api/shared/Amount.java#L10
It's already annotated with it